src/Commands/AddCommand.php

```diff
@@ -157,4 +166,17 @@ private function setGlobalGitHooksPath()
 
         $this->info("Global git hook path set to [{$globalHookDir}]");
     }
+
+    /**
+     * During a composer install or update process,
+     * a variable named COMPOSER_DEV_MODE will be added to the environment.
+     * If the command was run with the --no-dev flag,
+     * this variable will be set to 0, otherwise it will be set to 1.
+     *
+     * @see https://getcomposer.org/doc/articles/scripts.md#defining-scripts
+     */
+    private function isComposerDevMode()
+    {
+        return !$this->always && getenv('COMPOSER_DEV_MODE') === '1';
```

Comment:
I think we need to split this condition so we can have an error log for when `—always` is used.

Reply:
good idea. moved always check out